lib/tapioca/gemfile.rb

```diff
@@ -145,6 +145,32 @@ def contains_path?(path)
         end
       end
 
+      sig { returns(T::Array[String]) }
+      def exported_rbi_files
+        Dir.glob("#{full_gem_path}/rbi/*.rbi")
+      end
+
+      sig { returns(T::Boolean) }
+      def has_exported_rbi_files?
+        File.directory?("#{full_gem_path}/rbi")
+      end
+
+      sig { params(block: T.proc.params(arg: T::Array[RBI::Rewriters::Merge::Conflict]).void).returns(RBI::Tree) }
+      def exported_rbi_tree(&block)
+        rbi = RBI::Tree.new
+        rewriter = RBI::Rewriters::Merge.new
+        conflicts = []
+        exported_rbi_files.each do |file|
+          gem_rbi = RBI::Parser.parse_file(file)
+          rbi = RBI::Rewriters::Merge.merge_trees(rbi, gem_rbi, keep: RBI::Rewriters::Merge::Keep::NONE)
+          conflict = rewriter.merge(gem_rbi)
```

Comment:
Are we merging this thing twice?

`RBI::Rewriters::Merge.merge_trees` already calls `RBI::Rewriters::Merge.new` and `RBI::Rewriters::Merge#merge`.

I believe the whole method could be rewritten to something like this: 

```rb
      sig { returns(RBIMergeResult) }
      def exported_rbi_tree
        rewriter = RBI::Rewriters::Merge.new(keep: RBI::Rewriters::Merge::Keep::NONE)
        all_conflicts = []

        exported_rbi_files.each do |file|
          gem_rbi = RBI::Parser.parse_file(file)
          file_conflicts(rewriter.merge(gem_rbi))
          all_conflicts.concat(file_conflicts)
        end

        RBIMergeResult.new(tree: rewriter.tree, conflicts: conflicts)
      end
```

But looking at this I think I should just make `RBI::Rewriters` API better so it can handle multiple files merging and conflicts better.